### PR TITLE
Update file_control_MOD.js

### PR DESCRIPTION
**Pull request by Danno3817**
Added better error management for simpler diagnosis by the end user

### CODE REVIEW

**Comment by generalwrex** [comment]
That's why I put a Debug Mode toggle on my mods.

File: actions/file_control_MOD.js [hunk 1]
```diff
@@ -169,22 +170,32 @@ module.exports = {
             case 0:
             console.log("Create File Activated");
             try {
-                if (filePATH && fileNAME){
-                    fs.writeFileSync(FullFile,"");
+                if (filePATH){
+                    if (fileNAME) { 
+                        fs.writeFileSync(FullFile,"");
+                        console.log(`Sucessfully created: ${FullFile}`)
```
Comment (LasseNiermann) [inline actions/file_control_MOD.js]: Is that an error @Danno ?

**Comment by Danno3817** [comment]
should be fixed now

**Comment by Danno3817** [comment]
NP lol should have seen that from the get go haha!

**Pull request merged**